guake/callbacks.py

```diff
@@ -46,23 +46,6 @@ def on_find(self):
         # this is not implemented jet
         pass
 
-    def on_new_tab(self, *args):
-        self.notebook.new_page_with_focus(directory=self.terminal.get_current_directory())
```

Comment:
Sure it is not used by tab context menu ?

Reply:
Yes the tab context menu implements the on_new_tab method in https://github.com/Guake/guake/blob/master/guake/boxes.py#L365 there is no extra callback object (TabLabelEventBox passes it self as callback object). Since the tab context menu has only 3 on_* methods I decided to do it directly in TabLabelEventBox, maybe this should change in the future to be cleaner.

Reply:
oh, great. So this was a duplicate code :)